README.rst

```diff
@@ -53,35 +53,16 @@ You can also fork the CumulusCI-Test repository and use that as a demo since it
 
 Getting Started
 ---------------
+MetaCI can be run locally or configured for use on Heroku.
+For local setup see :ref: `running`.
 
-The recommended way to deploy and configure `MetaCI` is via the `MetaCI-CLI` project which can access your local CumulusCI project configurations including repo info and org configs to quickly get you up and running with `MetaCI`.
-
-Full documentation on launching and configuring `MetaCI` on Heroku using the `MetaCI-CLI`:
-http://metaci-cli.readthedocs.io/
-
-You can also reference the following session recordings showing live demos of how to configure `CumulusCI` and `MetaCI` for projects:
-https://www.youtube.com/watch?v=CgvYKDqb6Ng
-https://www.youtube.com/watch?v=TN-e785MqBA
+You can also reference `this session <https://www.youtube.com/watch?v=TN-e785MqBA>`_ from TrailheadDX '18 showing a live demo of how to configure `CumulusCI` and `MetaCI` for projects.
```

Comment:
This video uses the MetaCI-CLI commands that no longer work, so we probably should recommend it, at least not for the specific setup process.

Reply:
Oh, I know the first video did, but I didn't see any use of the MetaCI cli when I scanned this video. I can remove it too.

Reply:
I clicked around a bit and saw it at 24:58